js/script.js

```diff
@@ -0,0 +1,98 @@
+
+import {
+    $createButton,
+    $taskContainer
+} from './element.js'
+
+let isFormOpen = false
+let tasks = []
+
+const getUniqueId = () => Date.now();
+
+const makeForm = (taskInfo = {}) => {
+    let $form = document.createElement('li')
+    let $title = document.createElement('input');
+    let $addButton = document.createElement('button')
```

Comment:
```suggestion
    const $liElement = document.createElement('li')
    const $inputElement = document.createElement('input');
    const $addButton = document.createElement('button')
```

Html elements should be declared with const key words and it's better to use a name that is self explanatory.